README.md

```diff
@@ -4,79 +4,266 @@ Boost is a tool for Filecoin storage providers to manage data storage and retrie
+```
+[Libp2p]
+  ListenAddresses = ["/ip4/0.0.0.0/tcp/50000", "/ip6/::/tcp/0"]
+```
+
+15. Run Boost
+```
+boostd -vv run
+```
+
+Note the peer ID of the boost instance:
+```
+2022-06-10T09:32:28.819Z        INFO    boostd  boostd/run.go:114       Boost libp2p node listening     {"maddr": "{12D3KooWQNNWNiJ1mieEk9EHjDVF2qBc1FSjJGEzwjnMJzteApaW: [/ip4/172.17.0.2/tcp/50000 /ip4/127.0.0.1/tcp/50000]}"}
+```
+In this example: `12D3KooWQNNWNiJ1mieEk9EHjDVF2qBc1FSjJGEzwjnMJzteApaW`
 
+14. Set the peer ID and multi-address of the miner on chain
 ```
-export $(lotus auth api-info --perm=admin)
-export $(lotus-miner auth api-info --perm=admin)
+lotus-miner actor set-peer-id <peer id>
+lotus-miner actor set-addrs /ip4/127.0.0.1/tcp/50000
+```
+
+16. Open the Web UI
+
+Open http://localhost:8080 to see the Boost UI
 
-boostd --vv init \
-       --api-sealer=`lotus-miner auth api-info --perm=admin` \
-       --api-sector-index=`lotus-miner auth api-info --perm=admin` \
-       --wallet-publish-storage-deals=`lotus wallet new bls` \
-       --wallet-deal-collateral=`lotus wallet new bls` \
-       --max-staging-deals-bytes=50000000000
+### Make a deal with Boost
+
+1. Initialize the Boost client
+```
+boost init
 ```
 
-4. Run the Boost daemon service
+This will output the address of the wallet (it's safe to run the init command repeatedly).
 
+2. Send funds to the client wallet
+```
+lotus send --from=$DEFAULT_WALLET <client wallet> 10
 ```
-export $(lotus auth api-info --perm=admin)
 
-boostd --vv run
+3. Follow the guide at https://boost.filecoin.io/tutorials/how-to-store-files-with-boost-on-filecoin
+
+Note that above you already ran a command to export FULLNODE_API (and point it to your local devnet lotus daemon).
+
+Note also that the provider address is `t01000` and you will need to supply an appropriate `--storage-price` when using `boost deal` since the devnet has a minimum price. Alternatively, using "Settings" in the Boost web UI to set the deal price to zero.
+
+## Running Boost devnet in Docker
+
+### Building Docker images
+
+1. Select Lotus version, for example: `lotus_version=1.17.1-rc2`. It must be the tag name of [the Lotus git repo](https://github.com/filecoin-project/lotus/tags) without `v` prefix.
+
+2. Select Boost version, for example: `boost_version=1.3.0-rc1`. Docker images for Boost will be built on the current codebase. The `boost_version` is just used to tag images. If you want to build images for a specific boost version then you have to checkout that version first.
+
+3. Build images
+
+```
+cd docker
+make build/all lotus_version=1.17.1-rc2 boost_version=1.3.0-rc1
 ```
 
-5. Interact with Boost
+### Start devnet docker stack
 
-Pass the client address (wallet) and the provider address to the `dummydeal` command.
-Note that
-- the client address is the address of a wallet with funds in `lotus wallet list`
-- you can find the provider address in `~/.boost/config.toml` under the config key `Wallets.Miner`
+1. Run
```

Comment:
We need instruction, that a user must first `cd examples/devnet`

Reply:
Fixed. Also moved `examples/devnet` under `docker/devnet` -- I think the `docker-compose.yml` should be close to where we build the images.